### DISCUSSION TITLE
Using @automerge/automerge-wasm with importmap

### DISCUSSION THREAD

**Discussion by tomazy** [automerge/automerge#826] [Q&A]
Has anyone been able to use `@automerge/automerge-wasm` with [import maps](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap)?

Here's an example using [esm.sh](https://esm.sh/):

```html
<html>
<head>
  <title>Automerge wasm loading issue</title>
  <script type="importmap">
    {
      "imports": {
        "@automerge/automerge-wasm": "https://esm.sh/@automerge/automerge-wasm"
      }
    }
  </script>
</head>
<body>
<script type="module">
  import * as Automerge from "@automerge/automerge-wasm";
  const doc = Automerge.create();
  console.log(doc);
</script>
```
which gives following error:

```
automerge_wasm_bg.js:222 Uncaught TypeError: g.__wbindgen_add_to_stack_pointer is not a function
```
<img width="533" alt="Screenshot 2023-12-17 at 18 07 14" src="https://github.com/automerge/automerge/assets/967638/2f3d3753-f09f-4744-a810-28290079f31e">

It can be related to https://github.com/automerge/automerge/issues/555 and https://github.com/automerge/automerge/issues/788.

I wonder if I'm not loading it correctly or there's an issue with the package itself.

**Comment by ije** [comment]
i feel this is a bug of esm.sh not automerge, i will check
